app/serializers/current_user_serializer.rb

```diff
@@ -261,10 +261,6 @@ def is_anonymous
     object.anonymous?
   end
 
-  def reviewable_count
-    Reviewable.list_for(object).count
-  end
```

Comment:
Just want to confirm this is safe to remove even if the new notifications panel is disabled?

Reply:
Yes, it's safe - the serializer still includes the `reviewable_count` attribute, but it's now a method on `User` instead of the serializer.